charts/prompt-proto-service/templates/prompt-proto-service.yaml

```diff
@@ -75,6 +75,11 @@ spec:
           value: {{ .Values.imageNotifications.kafkaClusterAddress }}
         - name: RAW_MICROSERVICE
           value: {{ .Values.raw_microservice }}
```

Comment:
A counterexample to maybe explain the `with` block a bit better -- these two lines always define `RAW_MICROSERVICE`, but may define it as the empty string (which in this case means "there isn't one, don't try to connect").